package.json

```diff
@@ -11,6 +11,8 @@
     "eslint": "eslint index.js src",
     "eslint-fix": "eslint --fix index.js src",
     "flow": "flow",
+    "flow-coverage": "flow-coverage-report -i 'src/content/reducers/**/*.js' -t html -t text",
```

Comment:
`-t html` outputs an HTML report to `./flow-coverage`
`-t text` also outputs the report to your terminal